src/Integration/Persistence/ISolutionBinding.cs

```diff
@@ -0,0 +1,17 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="ISolutionBinding.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace SonarLint.VisualStudio.Integration.Persistence
+{
+    // Test only interface
+    internal interface ISolutionBinding
```

Comment:
Added for testability (to avoid creating real files and using the real credentials store)